packages/graph/hash_graph/lib/graph/src/store/postgres/ontology/read.rs

```diff
@@ -41,11 +50,17 @@ impl PersistedOntologyType for PersistedPropertyType {
     fn from_record(property_type: OntologyRecord<Self::Inner>) -> Self {
         let identifier = PersistedOntologyIdentifier::new(
             property_type.record.id().clone(),
-            property_type.account_id,
+            property_type.owned_by_id,
         );
+
         Self::new(
             property_type.record,
-            PersistedOntologyMetadata::new(identifier),
+            PersistedOntologyMetadata::new(
+                identifier,
+                property_type.created_by_id,
```

Comment:
Agreed, that's why in JS we define a `params` object which is passed to the methods. Reason I used this approach is because it's consistent with the remainder of the `new` method definitions in the rust code.

> Perhaps we should make a provenance module, and create new types for these ids

Happy to do this, although don't know how long it will take me as I haven't done this before and don't have a good sense how much it affects the diff.

Reply:
If the diff wasn't as big as it is, I would definitely block merging this PR to this amendment. Can this please be approached as a direct follow-up task and make sure future PRs are not touching TS when there are internal changes to Rust?

Reply:
[The task](https://app.asana.com/0/1202805690238892/1203193667254522/f) has been added to next week's sprint (after checking with Ciaran)

Reply:
> and make sure future PRs are not touching TS when there are internal changes to Rust?

I don't agree with this philosophy. It can be helpful to see a front-to-back change.

I think we should just be conscious about when a diff is becoming too big, and divide as makes sense. Not necessarily by service or language or whatever.